packages/dds/map/api-report/map.api.md

```diff
@@ -191,9 +191,9 @@ export interface IValueChanged {
 
 // @alpha
 export class LocalValueMaker {
-    constructor(serializer: IFluidSerializer);
+    constructor();
     fromInMemory(value: unknown): ILocalValue;
-    fromSerializable(serializable: ISerializableValue): ILocalValue;
+    fromSerializable(serializable: ISerializableValue, serializer: IFluidSerializer, bind: IFluidHandle): ILocalValue;
```

Comment:
Technically a breaking change to `@alpha`. But not caught by type tests due to a known issue. And thankfully in practice this class isn't used externally (and shouldn't be).

So it's fine, but awkward.

Reply:
> So it's fine

To clarify why it's fine - breaking changes are allowed in `main` now - but when action is requied on the consumer's side to digest the breaking change, we try to release the new alternative 1 release ahead of time. This does not apply here.